src/input_parsers/VnnLibParser.h

```diff
@@ -0,0 +1,75 @@
+/*********************                                                        */
+/*! \file VnnLibParser.cpp
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Idan Refaeli
+ ** This file is part of the Marabou project.
+ ** Copyright (c) 2017-2019 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved. See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** A parser for properties encoded in VNN-LIB format.
+
+**/
+
+#ifndef _VnnLibParser_h_
+#define _VnnLibParser_h_
+
+#include "InputQuery.h"
+#include "MString.h"
+#include "Vector.h"
+#include "Map.h"
+
+class VnnLibParser
+{
+    public:
```

Comment:
The indentation is off. "public:" and "private:" statements should not be indented. Please compare to other files